Build.bat

```diff
@@ -5,11 +5,11 @@ if '%1'=='-help' goto help
 if '%1'=='help' goto help
 if '%1'=='-h' goto help
 
-%~dp0vendor\tools\nuget.exe Install %~dp0vendor\packages.config -o %~dp0vendor\packages
-powershell -NoProfile -ExecutionPolicy Bypass -Command ^
-  "$psakeDir = ([array](dir %~dp0Vendor\packages\psake.*))[-1]; .$psakeDir\tools\psake.ps1 build.psake.ps1 %* -ScriptPath $psakeDir\tools ; if ($psake.build_success -eq $false) { exit 1 } else { exit 0 }"
+"%~dp0vendor\tools\NuGet.exe" install "%~dp0vendor\packages.config" -o "%~dp0vendor\packages"
+@SET cmd=$psakeDir = ([array](dir """%~dp0vendor\packages\psake.*"""))[-1]; ".$psakeDir\tools\psake.ps1" build.psake.ps1 %* -ScriptPath "$psakeDir\tools" ; if ($psake.build_success -eq $false) { exit 1 } else { exit 0 }
+powershell -NoProfile -ExecutionPolicy Bypass -Command ^ "%cmd%"
 goto :eof
 
 :help
-powershell -NoProfile -ExecutionPolicy Bypass -Command ^
-  "$psakeDir = ([array](dir %~dp0Vendor\packages\psake.*))[-1]; .$psakeDir\tools\psake.ps1 build.psake.ps1 -docs -ScriptPath $psakeDir\tools"
+@SET cmd=$psakeDir = ([array](dir """%~dp0vendor\packages\psake.*"""))[-1]; ".$psakeDir\tools\psake.ps1 build.psake.ps1" -docs -ScriptPath "$psakeDir\tools"
```

Comment:
I'm a little confused by the extra quotation marks around the argument to the dir command in this line (and the same on line 9).  Are they necessary?  Quick test seemed to work fine with just a single pair of quotes.

It's also not necessary to put the quotation marks around the PowerShell tokens such as $psakeDir, because they don't get expanded until after they're passed to a command, and spaces in those values don't matter.  (But they also don't hurt anything, so I'm fine with that part either way.)